pytx/pytx/vocabulary.py

```diff
@@ -5,7 +5,7 @@ class ThreatExchange(object):
     """
 
     URL = 'https://graph.facebook.com/'
-    VERSION = 'v2.6/'
+    VERSION = 'v2.8/'
```

Comment:
This is technically a breaking change, because v2.8 doesn't allow users to set the attack type and threat type. If we're going to do this, maybe we should wait until the 60 day time limit we gave people is up? And should we also bump the major version number since it's a breaking change? Not sure what versioning scheme we've decided to follow.